DeAceroDLabs/marketplace-frontend
# [DMP-504] Fix search request

## What is done

[[DMP-504] Fix Search request](https://deacero.atlassian.net/browse/DMP-504)

## How is done

1. Updated the configProxy, to replace the base url when a search request is made.
2. Fix a typo on the API send request method.

## How to test

1. Run `npm start`.
2. Login to the app.
3. Type something on the search bar.
4. Check that the request was made correctly. You should see the results of your search (if any) or a not found state.
